src/Glpi/Controller/Helpdesk/IndexController.php

```diff
@@ -54,19 +56,28 @@ public function __construct()
 
     #[SecurityStrategy(Firewall::STRATEGY_HELPDESK_ACCESS)]
     #[Route(
-        "/Home",
-        name: "glpi_selfservice_home",
+        "/Helpdesk",
+        name: "glpi_helpdesk_index",
+        methods: "GET"
+    )]
+    // Prevent old links from breaking
+    #[Route(
+        '/front/helpdesk.public.php',
+        name: "glpi_legacy_public_helpdesk",
```

Comment:
Is this really needed ? 
What do you think ?

Reply:
IMHO, we should add a dedicated listener for legacy URLs we want to be redirected to new corresponding URLs.
We will probably have to support some of them for a long time, and having a centralized list will probably be easier to maintain.

Reply:
So I'll remove it here and we backlog it somewhere ?